cg/apps/demultiplex/sample_sheet/index.py

```diff
@@ -25,7 +20,7 @@
 
 def is_dual_index(index: str) -> bool:
     """Determines if an index in the raw sample sheet is dual index or not."""
-    return "-" in index
+    return DASH in index
```

Comment:
Feels a bit over-the-top to have a constant for this? Also, made me think whether this should be considered a hyphen or a dash...

Reply:
Yes, this file dispensable
```
A dispensable is something pointless and unneeded whose absence would make the code cleaner, more efficient and easier to understand.
```

Reply:
It is a matter of taste, but it does read better.


This named constant is far less likely to be overlooked or misinterpreted than a raw '-', "-" might be.

Reply:
I do not agree that it is a dispensable as it makes the code easier to read and more abstract.

Reply:
I will leave it and discuss this in the coding guidelines

Reply:
Might be, I am a bit hesitant. But feel free to keep it as is.